packs/auth0.yml

```diff
@@ -5,6 +5,7 @@ PackDefinition:
   IDs:
     - Auth0.MFA.Policy.Disabled
     - Auth0.MFA.Risk.Assessment.Disabled
+    - Auth0.User.Invitation.Created
```

Comment:
I would swap lines 8 and 9 for alpha-sorting

Reply:
Fixed